pyproject.toml

```diff
@@ -43,6 +43,7 @@ myst-parser = { version = "^2.0.0", optional = true }
 sphinx = { version = "^7.2.5", optional = true }
 jinja2 = "^3.1.2"
 datasets = "^2.16.1"
+timm = "^0.9.12"
```

Comment:
who's timm?

Reply:
This was a merge from main from another PR relating to the detr model